src/test/php/lang/ast/unittest/emit/AnnotationSupport.class.php

```diff
@@ -11,138 +11,192 @@
  */
 abstract class AnnotationSupport extends EmittingTest {
 
+  /**
+   * Declares annotations, optionally including a type
+   *
+   * @param  string $declaration
+   * @return lang.reflection.Type
+   */
+  private function declare($declaration) {
+    return Reflection::type($this->type(
+      $declaration.(strstr($declaration, '<T>') ? '' : ' class <T> { }')
+    ));
+  }
```

Comment:
This should be moved to the parent class, it will aid the transition to using `xp-framework/reflection`!